wiki/osu!supporter/en.md

```diff
@@ -45,23 +41,34 @@ Laconic: Donation is used to sustain osu! overall
 | 12     | 26 (-46% max)                        |
 | 24     | 52 (-46% max)                        |
 
--   Costs are by individual account. There are no promotions for group gifting.
+Note:-
+
+- Costs are by per individual account. There are no promotions for group gifting.
 
 ### Decide
 
-By default, your account will be shown. If you do not want osu!supporter, click on "Gift another player" and write the name of the account. If you want to gift to more than one account, click "Add another tag" button beside the "Checkout" button. After confirming the account(s), drag the white button below an account to determine how long for the account and how much you will have to pay in total. You can set different durations for different accounts along with different cost. After confirming the duration and the accounts, click "Checkout".
+![An example of buying supporter](./img/sup_decide.jpg "An example of buying supporter")
+
+By default, only the player's account will be shown. If the player want to gift osu!supporter to another player, there is an option called `Gift another player` and type in the player's account name on the empty field provided. For multiple gifting, use `Add another tag` button beside the green `Checkout` button.
 
-![An example of buying supporter](O!s_Decide.jpg "An example of buying supporter")
+After the accounts are confirmed, drag the white slider below the account to determine osu!supporter active duration and the amount to pay for the account's osu!supporter. Each accounts can have different osu!supporter duration.
 
-Payment
--------
+Once the durations for all the accounts are confirmed, click on the green `Checkout` button to confirm the selection
 
-After confirming the duration and the accounts, click "Checkout". Here, you will be given various options. It is recommended to use [PayPal](https://www.paypal.com) or Credit Card services. There are other ways to pay (such as SMS or local service providers such as [Cherry Credits](https://www.cherrycredits.com/)), but additional fees will be charged for their effort. If you pay less than your "Total Cost" in "dorrah" (including additional cost if using non-Paypal), you will not receive your tag(s).
+### Payment
+
+Various payment methods will be displayed here.
+
+It is recommended to use [PayPal](https://www.paypal.com "PayPal") or `Credit Card` services. There are other ways to pay (such as `SMS` or local service providers such as [Cherry Credits](https://www.cherrycredits.com/ "Cherry Credits")) but additional fees may be charged.
```

Comment:
suggestion for the last part of the paragraph
`but additional fees may concur.` ... I think I used concur correctly 🤷‍♂️